quic/s2n-quic-transport/src/stream/stream_container.rs

```diff
@@ -145,10 +145,17 @@ struct InterestLists<S> {
     /// stream flow control window to increase
     waiting_for_stream_flow_control_credits:
         LinkedList<WaitingForStreamFlowControlCreditsAdapter<S>>,
+    transmission_counter: u8,
+    transmission_limit: u8,
```

Comment:
I think there might be a better place for these, since this struct is containing only lists